### PR TITLE
Added common components schema file.

### PR DESCRIPTION
**Pull request by V-R-Dighe**
Referencing component schemas from Components.yaml in each individual file.

### CODE REVIEW

File: openapiSpecifications/Users.yaml [hunk 1]
```diff
@@ -6,118 +6,6 @@ info:
...
-          example: "a_b_c_1"
-        param_errors:
-          type: array
-          items:
-            type: string
-          example: []
-
 paths:
   /v1/users/current:
     get:
```
Comment (AMAN-BARBARIA) [inline openapiSpecifications/Users.yaml:11]: current_user entity is not referenced via the common component

**Review by AMAN-BARBARIA — COMMENTED** [review]
1 review changes requested

File: openapiSpecifications/Users.yaml (hunk 1 above)
Comment (V-R-Dighe) [reply openapiSpecifications/Users.yaml:11]: Done @AMAN-BARBARIA.

**Review by AMAN-BARBARIA — APPROVED** [review]
LGTM

**Pull request merged**